src/main/groovy/wooga/gradle/paket/unity/internal/AssemblyDefinitionFileStrategy.groovy

```diff
@@ -19,4 +19,6 @@ package wooga.gradle.paket.unity.internal
 
 enum AssemblyDefinitionFileStrategy {
     manual, disabled
+
+    final static String fileExtension = "asmdef"
```

Comment:
That feels like an odd place for this static define.
Or better the name feels odd as it implies that the strategy has a file extension.

Reply:
Agreed; I have moved the constant